pkg/services/authn/authnimpl/sync/rbac_sync.go

```diff
@@ -117,6 +128,76 @@ func (s *RBACSync) fetchPermissions(ctx context.Context, ident *authn.Identity)
 	return permissions, nil
 }
 
+// translateK8sPermissions converts K8s-style permissions into Grafana's internal permission format.
+// It parses permissions in the format "group/resource:verb" and uses the mapper to translate them.
+func (s *RBACSync) translateK8sPermissions(ctx context.Context, k8sPerms []string) ([]accesscontrol.Permission, error) {
+	permissions := make([]accesscontrol.Permission, 0, len(k8sPerms))
+	for _, k8sPerm := range k8sPerms {
+		parts := strings.Split(k8sPerm, "/")
+		if len(parts) != 2 {
+			// Case group:*
+			if strings.HasSuffix(k8sPerm, ":*") {
+				group := strings.TrimSuffix(k8sPerm, ":*")
+				mappings := s.mapper.GetAll(group)
+				if len(mappings) == 0 {
+					s.log.Warn("No mappings found for group", "group", group)
+					continue
+				}
+
+				for _, mapping := range mappings {
+					actions := mapping.AllActions()
+					for _, action := range actions {
+						permissions = append(permissions, accesscontrol.Permission{Action: action})
```

Comment:
Permissions can be scoped as well. the permission registry allows to know which:
https://github.com/grafana/grafana/blob/7b66e7de438bff24c863667b6bc2af688c16a91b/pkg/services/authn/authnimpl/sync/rbac_sync.go#L100-L113